docs/ref/templates/builtins.txt

```diff
@@ -1054,8 +1054,11 @@ this example, the space around ``Hello`` won't be stripped::
 
 Outputs one of the syntax characters used to compose template tags.
 
-Since the template system has no concept of "escaping", to display one of the
-bits used in template tags, you must use the ``{% templatetag %}`` tag.
+The template system has no concept of “escaping” individual characters.
+However, you can use the {% templatetag %} tag to display one of the bits
+used in template tags.
+
+See also the `verbatim` tag for another way of including these characters.
```

Comment:
I'd add a link
```suggestion
See also the :ttag:`verbatim` tag for another way of including these characters.
```

Reply:
I am a bit unclear about the `:ttag:` syntax used here. Can you explain its purpose here?